hackerpet/__init__.py

```diff
@@ -0,0 +1,494 @@
+"""
```

Comment:
I typically like to see this kind of code broken out into separate module/files

Reply:
done